docs/migrate/rockset/index.md

```diff
@@ -63,7 +71,19 @@ Register now to learn about our migration services,
 and to have a live Q&A session with our experts.
 :::
 
+:::{card}
+:link: https://cratedb.com/resources/webinars/lp-wb-rockset-migration
+:link-alt: "Webinar Recordings"
+:class-header: sd-text-center sd-fs-5 sd-align-minor-center sd-font-weight-bold sd-text-capitalize
+:class-body: text-smaller
+:class-footer: text-smaller
+{material-outlined}`live_tv;2.7em`
+
+Watch recordings of previous sessions from this webinar series.
```

Comment:
Are recordings published at all? At least right now, I don't see any on the linked URL.

Reply:
Hi. I don't know about the exact procedure, because the content is behind a signup wall. What is actually behind when signing up?

Reply:
Once you submit the form you will be redirected to [this page](https://cratedb.com/resources/webinars/confirmation-rockset-to-cratedb) where you can watch the recording of our first webinar.

Reply:
Thanks for clarifying!